tools/snippet-testing/model/test_session_model.rb

```diff
@@ -1,4 +1,28 @@
+require 'yaml'
+
 module Model
   class TestSessionModel
+    attr_reader :testable, :source_folder, :output_folder, :root_output_folder, :relative_folder, :root_source_folder
+
+    alias testable? testable
+
+    def initialize(meta_yaml_path="test.yaml", parent_model = nil)
+      yaml_exists = File.file?(meta_yaml_path)
+      @source_folder = File.dirname(meta_yaml_path)
+      @root_source_folder = parent_model.nil? ? Dir.pwd : parent_model.root_source_folder
+      @relative_folder =@source_folder.sub(root_source_folder,"")
+      @testable = parent_model.nil? ? true : parent_model.testable
+      @root_output_folder = "testable_snippets"
```

Comment:
`testable_snippets` name should be defined only once. This is done in `tools/snippet-testing/snippet_tester.rb`